ngraph/test/backend/random_uniform.in.cpp

```diff
@@ -0,0 +1,340 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Tests now should be added to the template plugin (docs/template_plugin/tests/functional/op_reference/XXX) and not into the backend

Reply:
Please add reference tests to check reference implementation and SLT if we want to check plugins implementations.

Reply:
Applied this comment in separate PR: https://github.com/openvinotoolkit/openvino/pull/7012